Automattic/wp-calypso
# Premium themes: Remove one-off premium theme purchases (onboarding & theme showcase)

Related to https://github.com/Automattic/wp-calypso/issues/73265

## Proposed Changes

* Remove the theme being added in the checkout when the user selects the free option
* Renamed `Or start with a free site subheader` to `Or select another theme`
* Point the user to the Design Picker if they clicked on the free option


![](https://user-images.githubusercontent.com/1233880/237444788-c6abdd2d-ead2-45b6-8dff-06e1c5a1aa0d.png)

## Testing Instructions



* Use the live link from the comments
* Go to `/start/with-theme?ref=calypshowcase&theme=yuga&premium=true&theme_type=premium`
* On the plans page, make sure that the subheader text is now changed to `Or select another theme` (make sure it also changes on mobile ports)
* When you click on the free option, you are now pointed to the Design Picker (instead of the checkout page)
* Start the flow again and this time select a domain
* Make sure that you are pointed to the checkout page and only the domain is added in the cart (No `Premium theme: Yuga` item is added)
* Open an incognito page and check that we no longer display the pricing in the `Pick design` button.

## Pre-merge Checklist
